common/config/rush/browser-approved-packages.json

```diff
@@ -278,6 +278,10 @@
       "name": "@itwin/imodels-client-management",
       "allowedCategories": [ "integration-testing", "internal" ]
     },
+    {
+      "name": "@itwin/itwins-client",
+      "allowedCategories": [ "internal" ]
+    },
     {
```

Comment:
Remove `@itwin/projects-client` from `package.json` of standalone test-app and `browser-approved-packages.json`

Reply:
Done, thanks! I'll resolve this convo.